test/distributed/fsdp/test_fsdp_optim_state.py

```diff
@@ -142,8 +142,8 @@ def forward(self, x):
 class NestedModel(torch.nn.Module):
     def __init__(self) -> None:
         super().__init__()
-        self.block0 = BlockB(5, 7)
-        self.block1 = BlockB(7, 7)
+        self.block0 = BlockB(5, 3)
```

Comment:
Noob question: what's the difference between the two?

Reply:
With this new setting, we were able to reproduce the align padding issue.

Reply:
This does make me wonder if it would be worth utility to visualize/print the padding.

Reply:
Yup, since we are going to maintain FSDP FlatParameter version for a while, having simple utility to print the padding is going to be useful. We can make it a ramp-up task.